### ISSUE TITLE
Cannot find the type for custom attribute 'ValidateSpectreColor'

### ISSUE THREAD

**Issue by turpie** [ShaunLawrie/PwshSpectreConsole#3]
I'm getting a strange error if I run my script without previously running Start-SpectreDemo.

```
PowerShell 7.3.4
PS C:\Users\turpi> Import-Module PwshSpectreConsole
PS C:\Users\turpi> Write-SpectreFigletText "Hello!"
InvalidOperation: C:\Users\turpi\Documents\PowerShell\Modules\PwshSpectreConsole\0.2.0\public\PwshSpectreConsole.ps1:62
Line |
  62 |          [ValidateSpectreColor()]
     |          ~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find the type for custom attribute 'ValidateSpectreColor'. Make sure that the assembly that contains this
     | type is loaded.

```

However, if I run Start-SpectreDemo first my code works fine.

```
PS C:\Users\turpi> Write-SpectreFigletText "Hello!"
  _   _          _   _           _
 | | | |   ___  | | | |   ___   | |
 | |_| |  / _ \ | | | |  / _ \  | |
 |  _  | |  __/ | | | | | (_) | |_|
 |_| |_|  \___| |_| |_|  \___/  (_)

```

**Comment by ShaunLawrie** [comment]
I've fixed it in 0.2.1 if you try `Update-Module PwshSpectreConsole`. It turns out I didn't understand PowerShell classes, I still don't but I'm pretty sure it loads now...
Thanks for raising this 🙏

**Comment by turpie** [comment]
That's fixed it for me. Thank you.
